.github/workflows/load_test.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           AZURE_CLIENT_SECRET: ${{ secrets.AZURE_CLIENT_SECRET }}
           AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
           CORSO_LOAD_TESTS: true 
-          CORSO_LOG_FILE: stderr
+          CORSO_LOG_FILE: ./src/testlog/testlogging.log
```

Comment:
The path that gets uploaded here is different.
```suggestion
          CORSO_LOG_FILE: ./src/test_results/testlogging.log
```

Reply:
arg.  That's what I get for updating things during a meeting.  Good catch.